Demo/AppDelegate.swift

```diff
@@ -31,6 +31,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         window?.tintColor = Theme.tintColor
```

Comment:
move this line after applyTheme as tickMark doesn't show up for white theme if this is set 1st before applying the theme

Reply:
fixed